core/src/main/java/com/google/instrumentation/trace/TraceOptions.java

```diff
@@ -25,54 +29,86 @@
  */
 @Immutable
 public final class TraceOptions {
-  /**
-   * Bit to represent whether trace is sampled or not. If set, means this {@code Span} is part of a
-   * sampled trace and data should be exported to a persistent store.
-   */
-  public static final int IS_SAMPLED = 0x1;
-
+  // The size of TraceOptions in bytes.
```

Comment:
Are the various size constants needed? I think you can use (e.g.) Integer.SIZE instead.